src/unity/toolkits/object_detection/one_shot_object_detection/util/superposition.cpp

```diff
@@ -0,0 +1,128 @@
+/* Copyright © 2019 Apple Inc. All rights reserved.
+ *
+ * Use of this source code is governed by a BSD-3-clause license that can
+ * be found in the LICENSE.txt file or at https://opensource.org/licenses/BSD-3-Clause
+ */
+#include <boost/gil/extension/numeric/sampler.hpp>
+#include <boost/gil/extension/numeric/resample.hpp>
+
+#include <unity/toolkits/object_detection/one_shot_object_detection/util/mapping_function.hpp>
+#include <unity/toolkits/object_detection/one_shot_object_detection/util/superposition.hpp>
+#include <unity/toolkits/object_detection/one_shot_object_detection/util/quadrilateral_geometry.hpp>
+
+#define BLACK boost::gil::rgb8_pixel_t(0,0,0)
+#define WHITE boost::gil::rgb8_pixel_t(255,255,255)
+#define RGBA_WHITE boost::gil::rgba8_pixel_t(255,255,255,0)
```

Comment:
Avoid `#define` and macros when possible.

This can likely be defined as 
```
static const boost::gil::rgb8_pixel_t BLACK(0, 0, 0);
```
(Or use `constexpr` if that compiles)

Reply:
Fixed.